src/StatsigStore.ts

```diff
@@ -565,6 +565,13 @@ export default class StatsigStore {
     }
   }
 
+  public getGlobalEvaluationDetails(): EvaluationDetails {
+    return {
+      reason: this.reason ?? EvaluationReason.Uninitialized,
+      time: this.userValues.evaluation_time ?? Date.now(),
```

Comment:
If this function is for debugging purposes only, should we have it just return null in case it's absent?

Reply:
Yeah or 0